examples/ecs_with_ec2/main.tf

```diff
@@ -0,0 +1,100 @@
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
Please update the comment as this one does not fit the use case.

Reply:
I have updated it.